lrlex/src/lib/ctbuilder.rs

```diff
@@ -68,6 +68,15 @@ impl Visibility {
     }
 }
 
+/// Specifies the [^Rust Edition] that will be emitted during code generation.
```

Comment:
Is the `^` intentional?

Reply:
It was part of the footnote syntax of cargo doc, but looking at the generated doc (oops),
it looked terrible with the footnote style.  I managed to get it working so the link text shows up inline.
while not making line-length terrible asymmetric.

Reply:
There are still a couple of other `[^`s in the PR -- should they disappear too?

Reply:
Sorry still :sleeping: :coffee:, they probably should.

Reply:
Fixed the one in lrlex now in [931202f](https://github.com/softdevteam/grmtools/pull/367/commits/931202ffb74712dc6251ec8cebc9fa25adbcec56)